src/zosc.c

```diff
@@ -148,11 +148,11 @@ zosc_new (const char *address)
     zosc_t *self = (zosc_t *) zmalloc (sizeof (zosc_t));
     assert (self);
     //  Initialize class properties here
-    self->address = strdup(address);
-    self->format = strdup("");
+    self->format = "";
+    self->chunk = zchunk_new(address, strlen(address) + 1);
+    self->address = zchunk_data(self->chunk);
```

Comment:
src/zosc.c: In function 'zosc_new':
src/zosc.c:153:19: error: pointer targets in assignment differ in signedness [-Werror=pointer-sign]
     self->address = zchunk_data(self->chunk);